src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -647,9 +650,11 @@ export default class AircraftCommander {
         aircraft.setFlightPhase(FLIGHT_PHASE.TAKEOFF);
         aircraft.scoreWind('taking off');
 
-        readback.log = `wind ${roundedWindAngleInDegrees} at ${roundedWindSpeed}, runway ${runway.name}, ` +
+        readback.log = `wind ${roundedWindAngleInDegrees} degrees at ${roundedWindSpeed}, runway ${runway.name}, ` +
```

Comment:
no "degrees". In the absence of a feature to use multiple sets of phraseology, we use FAA phraseology as the standard, in which it is stated as `wind two one zero at one five gust two two`, so this should be reverted.